tests/test_plan.py

```diff
@@ -0,0 +1,143 @@
+from typing import Union, Optional
+
+import pytest
+
+import andi
+from andi import plan_str, FunctionArguments, CyclicDependencyError, \
+    NonProvidableError
+
+
+class A:
+    def __init__(self, e: 'E'): pass
```

Comment:
Could you please use full syntax here, with a new line? There is a high probability that either a future linter, or some future external contributors will raise an issue about not following pep8 :)

Reply:
Sure, done here: https://github.com/scrapinghub/andi/pull/8/commits/8155c0adc0e9623d1baf2e733dc424a27f55d904

Reply:
And here: https://github.com/scrapinghub/andi/pull/8/commits/6f93c945ec4b72392200d545ebd6bc34b9770c61